modules/oauth-api/src/main/java/org/mule/runtime/oauth/api/builder/AuthorizationCodeDanceCallbackContext.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.oauth.api.builder;
+
+import org.mule.runtime.oauth.api.AuthorizationCodeOAuthDancer;
+
+/**
+ * Implementations keep context information between the before and after callbacks set for an
+ * {@link AuthorizationCodeOAuthDancer}.
+ * 
+ * @since 4.0
+ */
+public interface AuthorizationCodeDanceCallbackContext {
```

Comment:
this interface is worthless as it is. It should have an `Optional<Object> getParameter(String)` or something of that sort.